### ISSUE TITLE
Button focus is not visible & Contrast ratio is less than 4.5 : 1 under high contrast theme

### ISSUE THREAD

**Issue by MIAeh** [SAP/fundamental-ngx#10105]
#### Is this a bug, enhancement, or feature request?
- Enhancement

#### Briefly describe your proposal.
It's a requirement from SAP Accessibility team.
They've advised that 

- no visible focus is available for `fd-button` element under high contrast theme.
<img width="452" alt="image" src="https://github.com/SAP/fundamental-ngx/assets/26197222/8ccd4580-da9b-4f89-827a-a36783b72863">

- In high contrast theme, the contrast ratio is less than the lowest bar: `4.5 : 1`

![image](https://github.com/SAP/fundamental-ngx/assets/26197222/edf6480f-1538-497a-b895-6836b8fed575)

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
- 0.36.3-rc.4

**Comment by droshev** [comment]
@MIAeh maybe there is something custom in your scenario. If you load our button [page](https://fundamental-ngx.netlify.app/#/core/button) in HCB you can see the button focus
<img width="299" alt="Screenshot 2023-07-03 at 8 48 47 PM" src="https://github.com/SAP/fundamental-ngx/assets/4380815/8943e9a3-ac39-4fe0-b8f6-ad425c0c990d">

**Comment by MIAeh** [comment]
> @MIAeh maybe there is something custom in your scenario. If you load our button [page](https://fundamental-ngx.netlify.app/#/core/button) in HCB you can see the button focus <img alt="Screenshot 2023-07-03 at 8 48 47 PM" width="299" src="https://user-images.githubusercontent.com/4380815/250698053-8943e9a3-ac39-4fe0-b8f6-ad425c0c990d.png">

Hi @droshev Thanks for your reply. I'm supposing that our tester is just changing the browser style instead of the theming switcher in fundamental. And in high-contrast browser theme, with the `emphasized` button in `Quartz`, the focus could be vague.

![Screenshot 2023-07-04 at 15 25 33](https://github.com/SAP/fundamental-ngx/assets/26197222/54fb8a25-740b-49c0-acf3-754633ca525d)

Do we consider this as an issue?

**Comment by droshev** [comment]
That doesn't look correct High-contrast. The buttons looks like
<img width="949" alt="Screenshot 2023-07-04 at 9 18 28 AM" src="https://github.com/SAP/fundamental-ngx/assets/4380815/cb8a0412-f10d-44b6-a533-0c171cafb652">
for me

**Comment by MIAeh** [comment]
@droshev  Hi deno, the tester is using the browser theme instead of fundamental theme. Hence, the button style will not listen to the expected fundamental styles.

**Comment by droshev** [comment]
@MIAeh then the user needs to switch to High Contrast themes first before using the OS theme, then there won't be any issue
